homeassistant/helpers/dispatcher.py

```diff
@@ -40,13 +77,15 @@ def async_dispatcher_connect(hass: HomeAssistantType, signal: str,
     if signal not in hass.data[DATA_DISPATCHER]:
         hass.data[DATA_DISPATCHER][signal] = []
 
-    hass.data[DATA_DISPATCHER][signal].append(target)
+    wrapped_target = wrap_callback(target)
```

Comment:
Can you update this PR to use the new log exception logic from the other PR

Reply:
Done.

Reply:
Very strange, I must have done some merge mistake..